test/loopback-dl.test.js

```diff
@@ -645,8 +645,28 @@ describe('DataSource define model', function() {
 
     var User = ds.define('User', {});
     assert.deepEqual(User.definition.properties.id,
-      {type: Number, id: 1, generated: true});
+      {type: Number, id: 1, generated: true, updateOnly: true});
+
+    done();
+  });
+
+  it('check forceId and getUpdateOnlyProperties', function(done) {
+    var ds = new DataSource('memory');
+    var Post = ds.define('Post', {
+      title: {type: String, length: 255},
+      date: {type: Date, default: function() {
+        return new Date();
+      }},
+    });
+
+    // check if forceId is added as true in ModelClass's settings[] explicitly, if id a generated (default) and forceId
+    // in from the model is true(unspecified is 'true' which is the default).
+    Post.settings.should.have.property('forceId').eql(true);
 
+    // check if method getUpdateOnlyProperties exist in ModelClass and check if the Post has 'id' in updateOnlyProperties list
```

Comment:
One assert per test please. There should be two tests: one checking `Post.settings.forceId`, the other verifying `getUpdateOnlyProperties()`. We may want to write more tests for `getUpdateOnlyProperties` in the future, it's probably a good idea to create a new `describe` block for tests of this method.